projects/canopy/src/lib/header/header-logo/header-logo.component.html

```diff
@@ -1,20 +1,14 @@
- <ng-container
-    *ngTemplateOutlet="
-      href ? linkTpl : imgTpl;
-      context: { img: { alt: alt, href: href, src: src } }
-    "
-  ></ng-container>
+<a *ngIf="href" [attr.href]="href" class="lg-header-logo__link" #logolink>
+  <img
+    [attr.alt]="alt"
+    [attr.src]="src"
+    [ngClass]="class"
+  />
+</a>
 
- <ng-template #linkTpl let-img="img">
```

Comment:
Why the hell did I make it so complicated last time 😂

Reply:
haha, I was looking at it thinking, there must be some reason for it but appears to just be a very Angular-y template-y solution to if/else